examples/terraform-aws-ssm-example/main.tf

```diff
@@ -0,0 +1,52 @@
+provider "aws" {
+  region = "us-east-2"
+}
+
+resource "aws_iam_role" "example" {
+  name = "example"
```

Comment:
Since we run tests in parallel, this name needs to include a variable to namespace it. The test code can then set that variable using `random.UniqueId` to ensure all the resources get unique names and don't clash.